src/views/components/tooltip/testid-tooltip/TestIdTooltip.js

```diff
@@ -2,32 +2,51 @@ import React, { Component } from 'react';
 import { Dimensions } from 'react-native-web';
 import { string, node } from 'prop-types';
 import Tooltip from '../Tooltip';
+import Storage from '../../../../utils/storage';
 
 class TestIdTooltip extends Component {
   static defaultProps = {
     id: 'test-id',
-  }
+  };
 
   static propTypes = {
     id: string,
     children: node,
+  };
+
+  // persist the showcodes on localstorgae
+  async persistCodes() {
+    const { children } = this.props;
+    let isDebugMode;
+
+    if ( typeof window === 'object' ) {
+      isDebugMode = window.originalQueryParams.showcodes;
```

Comment:
`showCodes` cammel case?

Reply:
it was originally all low case, i'm keeping the compatibility